resolwe_bio/processes/workflows/wes.yml

```diff
@@ -237,10 +242,9 @@
                 the user to replace all read groups in the INPUT file with a single new read group and assign all
                 reads to this read group in the OUTPUT BAM file. Addition or replacement is performed using
                 Picard's AddOrReplaceReadGroups tool. Input should take the form of -name=value delimited by a
-                \t, e.g. "-ID=1\t-PL=Illumina\t-SM=sample_1". See [tool's documentation](https://software
-                .broadinstitute.org/gatk/documentation/tooldocs/4.0.11.0/picard_sam_AddOrReplaceReadGroups.php) for
-                more information on tag names. Note that PL, LB, PU and SM are require fields. See caveats of
-                rewriting read groups in the documentation linked above.
+                \t, e.g. "-ID=1\t-PL=Illumina\t-SM=sample_1". See AddOrReplaceReadGroups documentation for more
+                information on tag names. Note that PL, LB, PU and SM are require fields. See caveats of rewriting read
```

Comment:
There are two main reasons I opted for not including the hyperlink:

* couldn't get the text to render properly on the frontend
* link will have to be updated every time we bump a docker version of the tool. This can get messy and if not maintained properly, can cause unnecessary amount of confusion.